### PR TITLE
Remove example folder

### PR DESCRIPTION
**Pull request by ngmachado**
This PR removes: 
- examples folder.
- Github Action that test examples apps.
- Remove Slack Bot notifications about examples

All examples are being migrated to [super-examples](https://github.com/superfluid-finance/super-examples)

### CODE REVIEW

**Comment by hellwolf** [comment]
@ngmachado I just did some small change in package.json. Please sync with devx team

**Comment by hellwolf** [comment]
Also, @kasparkallas wants [sdk-redux-examples](https://github.com/superfluid-finance/protocol-monorepo/tree/dev/examples/sdk-redux-examples) to be in packages instead. @ngmachado

**Comment by ngmachado** [comment]
@hellwolf @kasparkallas added redux examples to packages folder.

File: tasks/daily-slack-bot.js [hunk 1]
```diff
@@ -15,8 +15,6 @@ const topSectionMessage =
     "Looks like there are some lonely pull requests open in your area";
```
Comment (hellwolf) [inline tasks/daily-slack-bot.js:15]: @elvijsTDL you had a look of this?

**Comment by hellwolf** [comment]
@saflamini  any docs reference to the old path?

**Review by hellwolf — APPROVED** [review]

**Pull request merged**